spec/cypress/integration/schools/ChooseProgramme.feature

```diff
@@ -50,11 +47,9 @@ Feature: Induction tutors choosing programmes
     When I click on "link" containing "Continue"
     Then I should be on "school cohorts" page
     And the page should be accessible
-
-    When I click on "link" containing "2021"
-    Then I should be on "2021 school cohorts" page
-    And "page body" should contain "Add early career teachers"
-    And "page body" should not contain "Choose your training"
+    And "page body" should contain "Manage your training"
+    And "page body" should contain "Add your early career teacher and mentor details"
+    And percy should be sent snapshot called "Schools page"
```

Comment:
Please make the snapshot names unique - if the names are the same, one of them won't actually work (`"Schools page"` is the duplicate in this case)

Reply:
Now updated 👍